torch/testing/_internal/common_methods_invocations.py

```diff
@@ -246,6 +246,7 @@ def method_tests():
         ('log2', uniform_scalar(1e-2j, requires_grad=True), NO_ARGS, 'complex_scalar', (True,)),
         ('tanh', (S, S, S), NO_ARGS, '', (True,)),
         ('tanh', (), NO_ARGS, 'scalar', (True,)),
+        ('tanh', (S, S, S), NO_ARGS, 'complex', (True,)),
```

Comment:
you should include `tanh` in this [list](https://github.com/pytorch/pytorch/blob/7f11079769b0c9cb678a1046f1b0c9259ca3a3ac/test/test_autograd.py#L4193)

Reply:
Do you mean [here](https://github.com/pytorch/pytorch/blob/c06bd9289c9b73b200a4275a6ea6c34e8129ed4a/test/test_autograd.py#L4193)? I think I already included `tanh` in the list. Please correct me if I'm getting anything wrong here. Thanks!

Reply:
yeah sorry my b for some reason that change didn't show up for me last time